cicd/42c-conf-examples/mapping-and-discovery/42c-conf.yaml

```diff
@@ -1,13 +1,15 @@
-# This example shows how to map API files in your repository to APIs you already have
-# in another API collection in 42Crunch Platform.
+# This example shows how to map API files in your repository to APIs you
+# already have on 42Crunch Platform.
 #
-# When the CI/CD integration plugin starts, it begins with cleaning the 
-# API collection you have defined it to use. That is why the mapping 
-# must be done to another API collection.
+# When the CI/CD plugin starts, it creates an API collection
+# using the source control repository name and branch name.
 #
 # The plugin uploads any APIs it finds during the discovery phase
-# to 42Crunch Platform, so the APIs always get new API UUIDs. Mapping is useful 
-# if you want to preserve API UUIDs across separate audit runs.  
+# into this API collection. During the subsequent runs the collection
+# is kept is sync with the APIs discovered in your sources.
+#
+# However, if you already have APIs created on 42Crunch Platform and
```

Comment:
"However, if you already have APIs created on 42Crunch Platform and would like to keep these APIs in sync, you can use mapping." --> "If you have APIs in other API collections on 42Crunch Platform that you would like like to keep in sync with changes in the files in your source control, you can use mappings for this."